docs/guides/chia-gaming/gaming-users-guide.md

```diff
@@ -0,0 +1,110 @@
+---
+slug: /guides/gaming-users-guide
+title: Users Guide
+---
+
+:::important For Developers
+
+This user guide is intended to help developers test their gaming implementations. **Chia Network Inc will not host a version of this for users to play.** Before following this guide, developers must install and set up the gaming system. See the [Developers Guide](/guides/gaming-developers-guide) for installation and setup instructions.
```

Comment:
A couple notes here:
- Given that this is intended for developers, shouldn't it be called a developer's guide?
- A guide for end users would be nice to have as well. Of course this doesn't need to be added right away. The idea would be to give an overview of how the gaming platform will work, without the technical details.
- I didn't see any info in these docs about how the unroll in case of cheating will work. Maybe I missed it. It will be good to have that documented.

Reply:
This is effectively the users guide, there is a developers guide for developing but this overviews the processes that the players would take. Agreed on both other points, as for unroll we do not have that built into the app fully (and is associated with the shutdown with coins onchain issue so we don't want to emphasize that use case as of yet)